hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1907,6 +1907,15 @@
     </description>
   </property>
 
+  <property>
```

Comment:
Can you please use Java, annotation based configuration?

Reply:
This is done in the latest changeset